service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -21,11 +21,13 @@
 import static org.apache.commons.lang.StringUtils.capitalize;
 import static org.apache.commons.lang.StringUtils.uncapitalize;
 import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+import static uk.gov.hmcts.reform.fpl.service.DateFormatterService.formatLocalDateToString;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
+//TODO: refactor out dateFormatterService to provide static methods. Too many knock on effects to do within 1484
+// (FPLA-1482)
```

Comment:
I'm against leaving comments in the code, especially TODOs like this one. That's what tech debt tickets are for, it should all be described there.

Reply:
this will addressed in the next PR I have open